defaults/main.yml

```diff
@@ -1,11 +1,32 @@
-# Dont change this value, it's value is referenced
+
+# Tarsnap key contents
+# Should look like
+# TARSNAP_KEY: |
+#      # START OF TARSNAP KEY FILE
+#      contents
+#      # END OF TARSNAP KEY FILE
+TARSNAP_KEY: null
 TARSNAP_KEY_REMOTE_LOCATION: /root/tarsnap.key
-TARSNAP_KEY_LOCAL_TEMPLATE: null
 
-TARSNAPPER_CONF: null
+# You'll need to upload these scripts by yourself
+TARSNAP_BACKUP_PRE_SCRIPT: null
+TARSNAP_BACKUP_POST_SCRIPT: null
+
```

Comment:
An approach like for the nginx configuration, which deploys configuration files based on the VM hostname, wouldn't work? https://github.com/antoviaque/plebia-configuration/blob/da47d536bab30195541ab2650bb04a48626d8b81/roles/nginx/tasks/main.yml#L21-L22

Having the user uploading the scripts himself would be quite a pain to manage and update, I don't think we want to do this.